lms/templates/instructor/instructor_dashboard_2/send_email.html

```diff
@@ -3,50 +3,48 @@
-                </li>
-                <li>
-                    <input type="checkbox" name="send_to" value="staff" id="target_staff">
-                    <label for="target_staff">${_("Staff and Administrators")}</label>
-                </li>
-                <li>
-                    <input type="checkbox" name="send_to" value="learners" id="target_learners">
-                    <label for="target_learners">${_("All Learners")}</label>
-                </li>
-            </ul>
-            %if len(section_data['cohorts']) > 0:
-            <ul role="group" class="email-targets-secondary">
-                %for cohort in section_data['cohorts']:
-                <li>
-                    <input type="checkbox" name="send_to" value="cohort:${cohort.name}" id="target_cohort_${cohort.id}">
-                    <label for="target_cohort_${cohort.id}">
-                        ${_("Cohort: ") + cohort.name}
-                        %if cohort.name == section_data['default_cohort_name']:
-                        <br/>
-                        <div class="subheading">
-                            ${_("(Students without cohort assignment)")}
-                        </div>
-                        %endif
-                    </label>
-                </li>
-                %endfor
-            </ul>
-            %endif
-        </fieldset>
+            <fieldset>
+                <legend class="sr">${_("Send to:")}</legend>
+                <ul role="group" class="email-targets-primary">
+                    <li>
+                        <input type="checkbox" name="send_to" value="myself" id="target_myself">
+                        <label for="target_myself">${_("Myself")}</label>
+                    </li>
+                    <li>
+                        <input type="checkbox" name="send_to" value="staff" id="target_staff">
+                        <label for="target_staff">${_("Staff and Administrators")}</label>
+                    </li>
+                    <li>
+                        <input type="checkbox" name="send_to" value="learners" id="target_learners">
+                        <label for="target_learners">${_("All Learners")}</label>
+                    </li>
+                </ul>
+                %if len(section_data['cohorts']) > 0:
+                <ul role="group" class="email-targets-secondary">
+                    %for cohort in section_data['cohorts']:
+                    <li>
+                        <input type="checkbox" name="send_to" value="cohort:${cohort.name}" id="target_cohort_${cohort.id}">
+                        <label for="target_cohort_${cohort.id}">
+                            ${_("Cohort: ") + cohort.name}
+                            %if cohort.name == section_data['default_cohort_name']:
+                            <br/>
+                            <div class="subheading">
+                                ${_("(Students without cohort assignment)")}
```

Comment:
@catong I was intending to update this explanatory text, but missed it in my previous PR. What would be the best way to explain this? It's a cohort that may have been automatically created, if the following are all true:
- Cohorting is enabled for the course
- No automatic cohorts are created by staff
- A user without any cohort assignment access cohorted material

`Students without cohort assignment` is incorrect, because there may be users with a cohort of `None`, which is different. `Students with automatic assignment`, perhaps?
Further context: https://github.com/edx/edx-platform/pull/12602#discussion_r65573428